### PR TITLE
[stable/rabbitmq-ha] Updated metrics parameters

### PR DESCRIPTION
**Pull request by rstpv**
**What this PR does / why we need it**:

The current template uses the prometheus exporter port as the container port but, doesn't map the port inside the container (which is the 9090 currently) so, when you use a custom exporter port the mapping becomes incorrect.

I also added the following (Mostly changes in documentation): 

- Added the capabilities exporter value to allow custom specification (Default to bert and no_sort for better performance)
- Updated NOTES with info about how to access metrics depending on the type of service.
- Updated Readme as some of the values listed as default were incorrect (for example prometheus.exporter.enabled was false on default but was listed as true by default)
- Changed the default metrics port to 9419 because of a future breaking change in the rabbitmqexporter image (which recommends to use the 9419 port: https://github.com/kbudde/rabbitmq_exporter#breaking-change---100)

### CODE REVIEW

**Comment by rstpv** [comment]
Hello @sameersbn, do you have any comments on this? 
Thank you in advance.

**Comment by rstpv** [comment]
Hello can any of you please give feedback on this?
Thank you in advance.

/assign @sameersbn
/assign @linki 
/assign @scottrigby 
/assign @unguiculus

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.7.4
-version: 1.6.3
+version: 1.6.4
```
Comment (unguiculus) [inline stable/rabbitmq-ha/Chart.yaml]: Please bump minor version.

File: stable/rabbitmq-ha/values.yaml [hunk 2]
```diff
@@ -280,7 +280,9 @@ prometheus:
       pullPolicy: IfNotPresent
 
     ## Port Prometheus scrapes for metrics
-    port: 9090
+    port: 9419
```
Comment (unguiculus) [inline stable/rabbitmq-ha/values.yaml]: Why did you change the port?

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
cc @etiennetremel

File: stable/rabbitmq-ha/values.yaml (hunk 2 above)
Comment (rstpv) [reply stable/rabbitmq-ha/values.yaml]: @unguiculus The default port in the rabbitmq exporter docker image is 9090 (Which is why it was the default port) but, It'll change and, the default port will be 9419.
It might work with any of them (As the environment variable is being set in the change) but I think it might be better to reflect the true defaults. Should I set it back to 9090?

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (rstpv) [reply stable/rabbitmq-ha/Chart.yaml]: @unguiculus done.

File: stable/rabbitmq-ha/values.yaml (hunk 2 above)
Comment (unguiculus) [reply stable/rabbitmq-ha/values.yaml]: I'd stick with the default to avoid any confusion.

Comment (unguiculus) [reply stable/rabbitmq-ha/values.yaml]: I'd suggest you set it back. Otherwise this might cause confusion.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/rabbitmq-ha/values.yaml (hunk 2 above)
Comment (rstpv) [reply stable/rabbitmq-ha/values.yaml]: @unguiculus done

**Review by unguiculus — APPROVED** [review]
